extensions/stackdriver/metric/record.h

```diff
@@ -27,14 +27,18 @@ namespace Metric {
 void record(bool is_outbound, const ::Wasm::Common::FlatNode& local_node_info,
             const ::Wasm::Common::FlatNode& peer_node_info,
             const ::Wasm::Common::RequestInfo& request_info,
-            bool record_http_size_metrics);
+            bool record_http_size_metrics,
+            std::unordered_map<
```

Comment:
nit: make this const reference, and same to the function below.

Reply:
done.